### ISSUE TITLE
rz training should not use alpha

### ISSUE THREAD

**Issue by sahel-sh** [castorini/rank_llm#186]
This is a follow up of PR #175 
The training script for rz with use-logits=False should not use alpha for candidates identifiers.
